yunwoo/programmers-1/정수-제곱근-판별.js

```diff
@@ -0,0 +1,13 @@
+function solution(n) {
+  const squareRoot = Math.sqrt(n);
+
+  return Number.isInteger(squareRoot)
+    ? (squareRoot + 1) * (squareRoot + 1)
+    : -1;
+}
```

Comment:
```suggestion
function solution(n) {
  const squareRoot = Math.sqrt(n);

  return Number.isInteger(squareRoot)
    ? (squareRoot + 1) ** 2
    : -1;
}
```
- 고생하셨습니다 👍 제곱을 구할때는 ** 연산자를 이용한게 좀 더 깔끔해보입니다!

Reply:
- 오오! 제곱근 연산자를 생각못했네요.. 다음에는 고려해서 사용해보겠습니다 감사합니다🙇‍♂️